### ISSUE TITLE
Feature Request: Make config more "Craft-like"

### ISSUE THREAD

**Issue by fantasticmachine** [enupal/stripe#242]
Having separate handles for Test and Live keys is counter to the way Craft and Craft Plugins are normally configured.

It would be more "native" to be able to use a single set of handles and then assign values based on the environment - after all, that's why Environment Variables exist.

For instance, instead of:

```
'stripePayments' => [
	'testPublishableKey' => getenv('STRIPE_TEST_PUBLISHABLE_KEY'),
	'testSecretKey' =>  getenv('STRIPE_TEST_SECRET_KEY'),
	'testMode' => 1,
	'testWebhookSigningSecret' => getenv('STRIPE_TEST_WEBHOOK_SIGNING_SECRET'),
]
```

It would be possible to set

```
'stripePayments' => [
	'publishableKey' => getenv('STRIPE_PUBLISHABLE_KEY'),
	'secretKey' =>  getenv('STRIPE_SECRET_KEY'),
	'testMode' => getenv('STRIPE_TEST_MODE'),
	'webhookSigningSecret' => getenv('STRIPE_WEBHOOK_SIGNING_SECRET'),
]
```

The values for those variables would be different depending on my environment so there's no need to use different also use different handles.

For extra merit, being able to set other configuration options from a file would also be very useful.

Finally, it would also be nice if it were possible to set this up in a separate config file, `stripe-payments.php` rather than the `general.php` config.

**Comment by zizther** [comment]
Agreed, support for a plugin config file would be a nice way to manage settings with the use of ENV vars.
Is this something which would be supported @andrelopez
